npm/webpack-dev-server/CHANGELOG.md

```diff
@@ -3,64 +3,11 @@
 
 ### breaking
```

Comment:
@AtofStryker I don't think this breaking line should be here?

Reply:
it is supposed to be based on how we have release rc configured. Not a large fan of it but that's the current pattern